selfdrive/car/subaru/values.py

```diff
@@ -8,6 +8,7 @@ class CAR:
   ASCENT = "SUBARU ASCENT LIMITED 2019"
   IMPREZA = "SUBARU IMPREZA LIMITED 2019"
   FORESTER = "SUBARU FORESTER 2019"
+  LEGACY_2015 = "SUBARU LEGACY 2015 - 2018"
```

Comment:
Can you remove the model year?

Reply:
My current logic is to use model years for older models, I think 2019 or 2020 Legacy on Global platform will be without model year

Reply:
Let's make the convention this: global cars are just the model names and pre-global get a PREGLOBAL suffix

Reply:
done